index.html

```diff
@@ -37,6 +37,18 @@
       href: "https://www.w3.org/2025/02/council-report-fedid-dig-cred.html",
       date: "2025-02-20",
     },
+    "SUPPRESS": {
+      title: "The \"Segregate-and-Suppress\" Approach to Regulating Child Safety Online",
+      href: "https://papers.ssrn.com/sol3/papers.cfm?abstract_id=5208739",
+      date: "2025-04-08",
+      authors: ["Eric Goldman"],
+    },
+    "AGE-WS": {
+      title: "Upcoming: IAB/W3C Workshop on Age-Based Restrictions on Content Access",
+      href: "https://www.w3.org/news/2025/upcoming-iab-w3c-workshop-on-age-based-restrictions-on-content-access/",
```

Comment:
I weakly think it'd be better to link to the workshop itself, rather than its announcement.

```suggestion
      title: "IAB/W3C Workshop on Age-Based Restrictions on Content Access",
      href: "https://datatracker.ietf.org/group/agews/about/",
```